leakcanary-android/src/main/java/com/squareup/leakcanary/FragmentRefWatcher.java

```diff
@@ -0,0 +1,34 @@
+package com.squareup.leakcanary;
+
+import android.annotation.TargetApi;
+import android.app.Activity;
+import android.app.Fragment;
+import android.app.FragmentManager.FragmentLifecycleCallbacks;
+
+@TargetApi(26)
```

Comment:
consistency nit: the `O` named constant was used in the other file; maybe `O` here, too?